examples/request_reply_response/src/main/resources/application.yaml

```
@@ -3,9 +3,11 @@ spring:
     default-property-inclusion: NON_EMPTY
   cloud:
     function:
-      definition: responseToRequestSolace;responseMultiToRequestKnownSizeSolace;responseMultiToRequestRandomSizeSolace
+      definition: responseToRequestSolace;responseToLoggingSampleRequestSolace;responseMultiToRequestKnownSizeSolace;responseMultiToRequestRandomSizeSolace
```

this could be multi line, will make it easier to read
```suggestion
      definition: |
         responseToRequestSolace;
         responseToLoggingSampleRequestSolace;
         responseMultiToRequestKnownSizeSolace;
         responseMultiToRequestRandomSizeSolace
```